contained-innersource-enables-collaborative-product-development.md

```diff
@@ -0,0 +1,35 @@
+## Title  
+Contained inner source enables collaborative product development
+
+## Context
+* Multiple business units need a software component.
+* It makes sense for the business units to collaborate; each needs some different changes to the code.
+* One BU doesn't have the resources to do the development for all the rest.
+* InnerSource programs and tools exist.
+
+## Problem  
+The company needs to develop a software product component; multiple business units use this component.
+
+## Forces  
+* The product's importance to company revenue and the committed feature content and dates require a development paradigm that provides known, stable development resources (headcount).
+
```

Comment:
Maybe add a force describing the effect of _uncontained_ InnerSource on collaborative product development. I'm guessing you are thinking about added effort for soliciting and managing contributions?